lib/lightning_web/live/credential_live/json_schema_body_component.ex

```diff
@@ -83,12 +83,12 @@ defmodule LightningWeb.CredentialLive.JsonSchemaBodyComponent do
 
     type =
       case properties do
-        %{"format" => "uri"} -> :url_input
-        %{"type" => "string", "writeOnly" => true} -> :password_input
-        %{"type" => "string"} -> :text_input
-        %{"type" => "integer"} -> :text_input
-        %{"type" => "boolean"} -> :checkbox
-        %{"anyOf" => [%{"type" => "string"}, %{"type" => "null"}]} -> :text_input
+        %{"format" => "uri"} -> "url"
+        %{"type" => "string", "writeOnly" => true} -> "password"
+        %{"type" => "string"} -> "text"
+        %{"type" => "integer"} -> "text"
+        %{"type" => "boolean"} -> "checkbox"
+        %{"anyOf" => [%{"type" => "string"}, %{"type" => "null"}]} -> "text"
```

Comment:
Curious, why were these changed to strings?

Reply:
Because `NewInputs.input/1` expects a text on the `type` assign